### ISSUE TITLE
perGroupFunction not working according to README instructions

### ISSUE THREAD

**Issue by solsglasses** [dougmoscrop/serverless-plugin-split-stacks#78]
Hey @dougmoscrop, just a few hours ago you released `1.9.0` with a new feature `perGroupFunction`. I tried it according to the README, but it's not splitting my stack at all. Let me know what I'm doing wrong and I can help edit the README so others learn from my mistakes.

At the time I write this, the README states what I pasted below, but trying it (with an empty `stacks-map.js`) produces a single CF template with my 1300 😮 resources.

```
# ... In order to control the number of nested stacks, following configurations are needed:

  splitStacks:
    nestedStackCount: 20 # Controls the number of created nested stacks
    perFunction: false
    perType: false
    perGroupFunction: true
```

Do I need to add anything to `stacks-map.js`?

**Comment by dougmoscrop** [comment]
perGroupFunction has not been released yet, 1.9.0 has the feature for splitting by outputs, 1.10 will have perGroupFunction.  Optionally you can just install from git (in your package.json put 'github:dougmoscrop/serverless-plugin-split-stacks#sha-of-commit`)

**Comment by solsglasses** [comment]
Whoops, I was scoped to `master` branch while looking at the README. I thought I was on 1.9.0 release. IClosing issue, thanks
